pkg/storage/unified/sql/db/dbimpl/regression_incident_2144_test.go

```diff
@@ -16,93 +13,6 @@ import (
 const noIsolationLevelSupportErrStr = "sql: driver does not support non-" +
 	"default isolation level"
 
-var _ driver.Driver = driverWithoutIsolationLevel{}
```

Comment:
This section is moved to `driver_test.go` because it's also used in other parts, not only for the regression test. We should be able to delete the regression test file later.